src/utils/exchange.js

```diff
@@ -55,7 +66,34 @@ export const getProviderLogo = (provider?: string) => {
       return zeroxLogo;
     case PROVIDER_CHANGELLY:
       return changellyLogo;
+    case PROVIDER_MOONPAY:
+      return moonPayLogo;
+    case PROVIDER_SENDWYRE:
+      return sendWyreLogo;
     default:
       return '';
   }
 };
+
+export const checkFiatProvider = (provider: string) => {
```

Comment:
The word "check" is ambiguous, I'd rather name this something like `isFiatProvider`, for example.

Reply:
Updated in last commit 👍